src/System.Buffers.Primitives/System/Buffers/ReaderWriter/BufferReader_binary.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Buffers.Binary;
+using System.Collections.Sequences;
+
+namespace System.Buffers
+{
+    public ref partial struct BufferReader<TSequence> where TSequence : ISequence<ReadOnlyMemory<byte>>
+    {
+        public bool TryRead(out int value, bool littleEndian = false)
```

Comment:
Can we not add these APIs back? I don't think they are up to the quality bar and they shouldn't be part of the API review